README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
AFAIK it is important only when you are using older licenses. E.g. MIT or GPLv2, but Apache 2.0 explicitly defines that you need to have the mentioned rights when you do a `Contribution` [1].

It might be that a DCO could give some extra legal security, but I think there is no prove that it actually does (maybe you know of one). Now one can argue, that you are not necessarily aware of the license of the project you are contributing to, but that also applies to just adding the `-s` to the git message, like @admiyo pointed out. Having the `LICENSE` file in a well known place, should normally be good enough.

However, if you can point to resources which show that this is really a requirement or a definitive improvement, then we should add it.

[1] https://github.com/kubevirt/kubevirt/blob/master/LICENSE#L49